### ISSUE TITLE
Issue when dump servers with versions above 6.18

### ISSUE THREAD

**Issue by marcusricardoaguiar** [pgadmin-org/pgadmin4#6137]
I am having the following issue when trying to run dump-servers with version above 6.18:

**Using version 6.18:**
![image](https://user-images.githubusercontent.com/8216333/232150740-ff7f33e2-72e3-4bf8-858d-cd8f954ef2a8.png)

**Using version 6.19:**
![image](https://user-images.githubusercontent.com/8216333/232151153-354d35c6-1bd3-4668-84e4-2a87965518b0.png)

**Using version 6.20:**
![image](https://user-images.githubusercontent.com/8216333/232151506-64a9f651-f714-4bd6-aa5a-33d9df3c3c89.png)

**Using version 6.21:**
![image](https://user-images.githubusercontent.com/8216333/232151851-24574000-19f8-424e-82d4-938c4921baad.png)

As shown in the first screen, this configuration doesn't have any server in pgadmin4.
Other than this, the pgadmin4 is install in centos, version 8.

**Comment by akshay-joshi** [comment]
@marcusricardoaguiar 

Can we have a screen sharing session to resolve this issue?

**Comment by marcusricardoaguiar** [comment]
> @marcusricardoaguiar
> 
> Can we have a screen sharing session to resolve this issue? It's not reproducible to us

Yes, Sure. I'm available to the session. :smile:

**Comment by marcusricardoaguiar** [comment]
Great. What about 6 PM IST?

Thanks!

**Comment by akshay-joshi** [comment]
Join Zoom Meeting
[https://enterprisedb.zoom.us/j/95616231746?pwd=bnNXcWlTbDc2aWEydGR2RHpXak9Xdz09](https://www.google.com/url?q=https://enterprisedb.zoom.us/j/95616231746?pwd%3DbnNXcWlTbDc2aWEydGR2RHpXak9Xdz09&sa=D&source=calendar&ust=1682162167319936&usg=AOvVaw3JXuCWsIC9R-lojhyb5BNU)

Meeting ID: 956 1623 1746
Passcode: 659986

**Comment by marcusricardoaguiar** [comment]
Hi @akshay-joshi , I've tried again as we discussed. I removed and reinstall the pgadmin4, I've also removed the database file and tried again. However, It still doesn't work as expected. Even though, the error remains. Do you have any other ideas about what I can do to fix it? :thinking:

**Comment by akshay-joshi** [comment]
I would suggest using Import/Export servers from GUI as it is working fine.  Or we can have another screen-sharing session to check why the new code failed. Meanwhile I'll try to figure out the problem

**Comment by yogeshmahajan-1903** [comment]
@marcusricardoaguiar 

We could reproduce the issue and now fixed it. This will be available in 7.1 release. by the time could you please update below in ~/web/pgadmin/utils)/__init__.py file in function dump_database_servers in first try-except block - 

```
file_path = filename_with_file_manager_path(
            unquote(output_file), skip_permission_check=from_setup)
```
Change above lines to - 
```
if from_setup:
        file_path = unquote(output_file)
else:
       file_path = filename_with_file_manager_path(unquote(output_file))
```

And try dump_servers again?

**Comment by marcusricardoaguiar** [comment]
Hi @yogeshmahajan-1903 ,

thank you so much for that. I've been using version 6.19. I've update the file `/usr/pgadmin4/web/pgadmin/utils/__init__.py` as you suggested and it worked pretty fine.

Thanks again!

**Comment by marcusricardoaguiar** [comment]
FYI: I had the same issue with `load-servers` and I had to update the function `load_database_servers` as you suggested to make it work.

**Comment by yogeshmahajan-1903** [comment]
@marcusricardoaguiar 

Yeah. --load_servers related issue was fixed in 6.20 as part of #5751. As you are still using 6.19 you are facing the issue.
Requesting you to upgrade latest version 7.0

**Comment by akshay-joshi** [comment]
Closing this as it works for the user.
